### ISSUE TITLE
Incomplete FEA languagesystem statements

### ISSUE THREAD

**Issue by lianghai** [simoncozens/fontFeatures#54]
https://github.com/simoncozens/fontFeatures/blob/f31cdadf7aa3eae1fb3953b692840ef3fc04c82b/fontFeatures/feaLib/FontFeatures.py#L13-L14

Is this `< 2` assuming a `DFLT dflt` is always present next to a non-default language system based on the commented code below?

https://github.com/simoncozens/fontFeatures/blob/f31cdadf7aa3eae1fb3953b692840ef3fc04c82b/fontFeatures/__init__.py#L216-L220

**Comment by simoncozens** [comment]
Yeah, this is probably a mistake - the idea was that if you’re not doing anything involving language-specific rules then you don’t need a languagesystem statement and the implicit dflt/DFLT is enough.

**Comment by lianghai** [comment]
How about we make it explicit in FEA? I quite like how the result from `.asFeaAST` already tries to be explicit about script/language statements and lookup flags.

Did you comment out the `DFLT` insertion logic in `fontFeatures.__init__` because you realized the concept of `DFLT` is specific to OTL? We can probably move it to `.feaLib.FontFeatures.add_language_system_statements`, and make `.add_language_system_statements` always generate languagesystem statements?

**Comment by simoncozens** [comment]
The code which generates FEA has undergone a bit of a transformation. Originally it was intended to be maximally human readable; the idea being that the output from otf2fea should be basically the way a human would write it - eg non-shared lookups embedded inside feature statements, etc.

Recently it has moved towards being maximally correct/robust even if less readable. But this bit of the code didn’t catch up. We should make the languagesystem statements explicit.
